http/analysis-service/server.py

```diff
@@ -238,8 +235,22 @@ def update_configs(jobs=None):
             "job_data_bucket": job['data_bucket'],
             "job_output_dir": job['output_dir']
         }
-        #FIXME: find a better way than embedding the path all over the place.
-        filename = "/home/ubuntu/telemetry_analysis/jobs/{0}.json".format(job['id'])
+
+    return config
+
+def update_configs(jobs=None):
+    if jobs is None:
+        jobs = []
+        for j in get_jobs():
+            jobs.append(j)
+
+    for job in jobs:
+        # Should be either:
```

Comment:
This comment doesn't make sense here - it's about the `iam_role` variable in a non-cluster context.